### PR TITLE
Switch to github.com/pkg/errors

### PR DESCRIPTION
**Pull request by poothia**
part of https://github.com/kubernetes-sigs/federation-v2/issues/381#issue-376377377

/sig multicluster

### CODE REVIEW

File: examples/slackcontroller/slackcontroller.go [hunk 1]
```diff
@@ -23,8 +23,10 @@ import (
 	"time"
 
 	"github.com/golang/glog"
+	"github.com/pkg/errors"
+
 	corev1 "k8s.io/api/core/v1"
-	"k8s.io/apimachinery/pkg/api/errors"
+	api_errors "k8s.io/apimachinery/pkg/api/errors"
```
Comment (gyliu513) [inline examples/slackcontroller/slackcontroller.go]: s/api_errors/apierrors

This can keep consistent with Kubernetes source code.

Comment (poothia) [reply examples/slackcontroller/slackcontroller.go]: @gyliu513 done.

**Pull request merged**